### PR TITLE
Enable testing with testsuite

### PR DESCRIPTION
**Pull request by legion151**
Using version 2.x of OeC and allows performing the tests with the tessuite

### CODE REVIEW

File: app/src/main/java/org/openecard/demo/activities/UseCaseSelectorActivity.java [hunk 1]
```diff
@@ -52,40 +46,30 @@
...
  */
-public class UseCaseSelectorActivity extends AppCompatActivity {
+public class UseCaseSelectorActivity extends FragmentActivity {
 
 	private static final Logger LOG = LoggerFactory.getLogger(UseCaseSelectorActivity.class);
 
-	private static final String DEFAULT_TC_TOKEN_URL = "https://test.governikus-eid.de:443/Autent-DemoApplication/RequestServlet;?provider=demo_epa_20&redirect=true";
-	//private static final String DEFAULT_TC_TOKEN_URL = "https://test.governikus-eid.de:443/Autent-DemoApplication/RequestServlet;?provider=demo_epa_can&redirect=true";
+//	private static final String TEST_SERVICE_URL = "https://test.governikus-eid.de:443/Autent-DemoApplication/RequestServlet;?provider=demo_epa_20&redirect=true";
+	//private static final String TEST_SERVICE_URL = "https://test.governikus-eid.de:443/Autent-DemoApplication/RequestServlet;?provider=demo_epa_can&redirect=true";
```
Comment (Adelrisk) [inline app/src/main/java/org/openecard/demo/activities/UseCaseSelectorActivity.java]: I don't think we need these.

File: app/src/main/java/org/openecard/demo/activities/UseCaseSelectorActivity.java [hunk 2]
```diff
@@ -105,109 +89,78 @@ protected void onCreate(Bundle savedInstanceState) {
...
-			LOG.warn(msg, ex);
-			throw new RuntimeException(msg, ex);
+	public void onStart() {
+		super.onStart();
+		Button btnPinMgmt = findViewById(R.id.btnPinManagement);
+		if(btnPinMgmt!=null) {
+			btnPinMgmt.setOnClickListener(v -> {
+				pinManagement();
+			});
 		}
```
Comment (Adelrisk) [inline app/src/main/java/org/openecard/demo/activities/UseCaseSelectorActivity.java:97]: Doesn't the respective fragment do this?

File: app/src/main/java/org/openecard/demo/fragments/PINBlockedFragment.java [hunk 3]
```diff
@@ -44,22 +38,22 @@
...
+       //     activity.cancelEacGui();
+       //     activity.setStartPinManagementDialog(() -> {
+       //         Intent i = new Intent(Intent.ACTION_VIEW);
+       //         i.setClass(getActivity(), PINManagementActivity.class);
+       //         i.setData(Uri.parse("/eID-Client?ShowUI=PINManagement"));
+       //         i.putExtra(ActivationImplementationInterface.RETURN_CLASS, UseCaseSelectorActivity.class.getName());
+       //         activity.startActivity(i);
+       //         activity.finish();
+       //     });
+       // });
```
Comment (Adelrisk) [inline app/src/main/java/org/openecard/demo/fragments/PINBlockedFragment.java:56]: Time to remove this. :wink:

**Review by Adelrisk — COMMENTED** [review]
Generally looks good. (Review delayed because of unfamiliarity with the tool.)

Minor clean-up below.

**Comment by legion151** [comment]
I am cleaning the app atm - unfortunately i didn't read those comments before. 
I will close this and create a new PR when done.

**Pull request closed without merging**